index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>"Hello Microverse!"</h1>
```

Comment:
Great job @AbhishekkM05 🤝

- [ ] As stated in the project requirements, kindly add a class to your `h1` and apply your styles to this class instead of applying the styles to the HTML element `h1`.